src/commands/dev/mod.rs

```diff
@@ -29,8 +29,6 @@ pub fn dev(
     // before serving requests we must first build the Worker
     build_target(&target)?;
 
-    deploy::pre_upload()?;
-
```

Comment:
Nice

Reply:
I'm still iffy on whether or not it would have been nicer to keep this, or to do all the hackery I did to get a GlobalUser to pass into `api_migrations()`.